src/index.ts

```diff
@@ -158,7 +158,7 @@ class SimpleMetrics implements Metrics, Startable {
         }
       }
 
-      this.onMetrics(output)
+      this.onMetrics(JSON.parse(JSON.stringify(output)))
```

Comment:
why not use https://developer.mozilla.org/en-US/docs/Web/API/structuredClone ?

Reply:
No reason - PR please!

Reply:
That said, JSON stringify/parse seems to be faster? - [benchmark](https://jsbenchmark.com/#eyJjYXNlcyI6W3siaWQiOiJ5UWxIeW55Tjl1N01EYjFQRjZyaUUiLCJjb2RlIjoiSlNPTi5wYXJzZShKU09OLnN0cmluZ2lmeSh7XG4gIFwibGlicDJwX3RyYW5zcG9ydF9tYW5hZ2VyX2xpc3RlbmVyc1wiOiA1LFxuICBcImxpYnAycF9kaWFsbGVyX3BlbmRpbmdfZGlhbHNcIjogMCxcbiAgXCJsaWJwMnBfZGlhbGxlcl9pbl9wcm9ncmVzc19kaWFsc1wiOiAzNSxcbiAgXCJsaWJwMnBfd2VicnRjX2RpYWxlcl9ldmVudHNfdG90YWxcIjoge30sXG4gIFwibGlicDJwX3dlYnJ0Y19saXN0ZW5lcl9ldmVudHNfdG90YWxcIjoge30sXG4gIFwibGlicDJwX3dlYnJ0Yy1kaXJlY3RfZGlhbGVyX2V2ZW50c190b3RhbFwiOiB7fSxcbiAgXCJsaWJwMnBfd2VidHJhbnNwb3J0X2RpYWxlcl9ldmVudHNfdG90YWxcIjoge1xuICAgIFwicGVuZGluZ1wiOiA1MyxcbiAgICBcInJlYWR5X2Vycm9yXCI6IDUsXG4gICAgXCJyZWFkeVwiOiAxNCxcbiAgICBcIm9wZW5cIjogMTNcbiAgfSxcbiAgXCJpcGZzX2JpdHN3YXBfc3RhdHNfcGVlcnNcIjogMCxcbiAgXCJpcGZzX2JpdHN3YXBfbGVkZ2VyX21hcFwiOiAxLFxuICBcImlwZnNfYml0c3dhcF93YW50X21hbmFnZXJfcGVlcnNcIjogMTgsXG4gIFwiaXBmc19iaXRzd2FwX3dhbnRsaXN0XCI6IDAsXG4gIFwibGlicDJwX2Nvbm5lY3Rpb25fbWFuYWdlcl9jb25uZWN0aW9uc1wiOiB7XG4gICAgXCJpbmJvdW5kXCI6IDAsXG4gICAgXCJvdXRib3VuZFwiOiAxOFxuICB9LFxuICBcImxpYnAycF9wcm90b2NvbF9zdHJlYW1zX3RvdGFsXCI6IHtcbiAgICBcIm91dGJvdW5kIHVubmVnb3RpYXRlZFwiOiA0LFxuICAgIFwib3V0Ym91bmQgL2xpYnAycC9jaXJjdWl0L3JlbGF5LzAuMi4wL2hvcFwiOiAxLFxuICAgIFwib3V0Ym91bmQgL21lc2hzdWIvMS4xLjBcIjogNCxcbiAgICBcImluYm91bmQgL21lc2hzdWIvMS4xLjBcIjogNCxcbiAgICBcImluYm91bmQgL2lwZnMvYml0c3dhcC8xLjIuMFwiOiAxXG4gIH0sXG4gIFwibGlicDJwX2Nvbm5lY3Rpb25fbWFuYWdlcl9wcm90b2NvbF9zdHJlYW1zX3Blcl9jb25uZWN0aW9uXzkwdGhfcGVyY2VudGlsZVwiOiB7XG4gICAgXCJvdXRib3VuZCB1bm5lZ290aWF0ZWRcIjogNCxcbiAgICBcIm91dGJvdW5kIC9saWJwMnAvY2lyY3VpdC9yZWxheS8wLjIuMC9ob3BcIjogMSxcbiAgICBcIm91dGJvdW5kIC9tZXNoc3ViLzEuMS4wXCI6IDEsXG4gICAgXCJpbmJvdW5kIC9tZXNoc3ViLzEuMS4wXCI6IDEsXG4gICAgXCJpbmJvdW5kIC9pcGZzL2JpdHN3YXAvMS4yLjBcIjogMVxuICB9LFxuICBcImxpYnAycF9rYWRfZGh0X3dhbl9ydW5uaW5nX3F1ZXJpZXNcIjogMyxcbiAgXCJsaWJwMnBfa2FkX2RodF93YW5fcXVlcnlfdGltZV9zZWNvbmRzXCI6IDI0OCxcbiAgXCJsaWJwMnBfa2FkX2RodF93YW5fcm91dGluZ190YWJsZV9zaXplXCI6IDg1LFxuICBcImxpYnAycF9rYWRfZGh0X3dhbl9waW5nX3F1ZXVlX3NpemVcIjogOTQsXG4gIFwibGlicDJwX2thZF9kaHRfd2FuX3BpbmdfcnVubmluZ1wiOiAxMCxcbiAgXCJsaWJwMnBfa2FkX2RodF9sYW5fcnVubmluZ19xdWVyaWVzXCI6IDMsXG4gIFwibGlicDJwX2thZF9kaHRfbGFuX3F1ZXJ5X3RpbWVfc2Vjb25kc1wiOiA2MTMsXG4gIFwibGlicDJwX2thZF9kaHRfbGFuX3JvdXRpbmdfdGFibGVfc2l6ZVwiOiA2MyxcbiAgXCJsaWJwMnBfa2FkX2RodF9sYW5fcGluZ19xdWV1ZV9zaXplXCI6IDAsXG4gIFwibGlicDJwX2thZF9kaHRfbGFuX3BpbmdfcnVubmluZ1wiOiAwXG59KSkiLCJuYW1lIjoianNvbiJ9LHsiaWQiOiJ4WDd1Z1pXaG9XYXJ1NHVQU1FZd3MiLCJjb2RlIjoic3RydWN0dXJlZENsb25lKHtcbiAgXCJsaWJwMnBfdHJhbnNwb3J0X21hbmFnZXJfbGlzdGVuZXJzXCI6IDUsXG4gIFwibGlicDJwX2RpYWxsZXJfcGVuZGluZ19kaWFsc1wiOiAwLFxuICBcImxpYnAycF9kaWFsbGVyX2luX3Byb2dyZXNzX2RpYWxzXCI6IDM1LFxuICBcImxpYnAycF93ZWJydGNfZGlhbGVyX2V2ZW50c190b3RhbFwiOiB7fSxcbiAgXCJsaWJwMnBfd2VicnRjX2xpc3RlbmVyX2V2ZW50c190b3RhbFwiOiB7fSxcbiAgXCJsaWJwMnBfd2VicnRjLWRpcmVjdF9kaWFsZXJfZXZlbnRzX3RvdGFsXCI6IHt9LFxuICBcImxpYnAycF93ZWJ0cmFuc3BvcnRfZGlhbGVyX2V2ZW50c190b3RhbFwiOiB7XG4gICAgXCJwZW5kaW5nXCI6IDUzLFxuICAgIFwicmVhZHlfZXJyb3JcIjogNSxcbiAgICBcInJlYWR5XCI6IDE0LFxuICAgIFwib3BlblwiOiAxM1xuICB9LFxuICBcImlwZnNfYml0c3dhcF9zdGF0c19wZWVyc1wiOiAwLFxuICBcImlwZnNfYml0c3dhcF9sZWRnZXJfbWFwXCI6IDEsXG4gIFwiaXBmc19iaXRzd2FwX3dhbnRfbWFuYWdlcl9wZWVyc1wiOiAxOCxcbiAgXCJpcGZzX2JpdHN3YXBfd2FudGxpc3RcIjogMCxcbiAgXCJsaWJwMnBfY29ubmVjdGlvbl9tYW5hZ2VyX2Nvbm5lY3Rpb25zXCI6IHtcbiAgICBcImluYm91bmRcIjogMCxcbiAgICBcIm91dGJvdW5kXCI6IDE4XG4gIH0sXG4gIFwibGlicDJwX3Byb3RvY29sX3N0cmVhbXNfdG90YWxcIjoge1xuICAgIFwib3V0Ym91bmQgdW5uZWdvdGlhdGVkXCI6IDQsXG4gICAgXCJvdXRib3VuZCAvbGlicDJwL2NpcmN1aXQvcmVsYXkvMC4yLjAvaG9wXCI6IDEsXG4gICAgXCJvdXRib3VuZCAvbWVzaHN1Yi8xLjEuMFwiOiA0LFxuICAgIFwiaW5ib3VuZCAvbWVzaHN1Yi8xLjEuMFwiOiA0LFxuICAgIFwiaW5ib3VuZCAvaXBmcy9iaXRzd2FwLzEuMi4wXCI6IDFcbiAgfSxcbiAgXCJsaWJwMnBfY29ubmVjdGlvbl9tYW5hZ2VyX3Byb3RvY29sX3N0cmVhbXNfcGVyX2Nvbm5lY3Rpb25fOTB0aF9wZXJjZW50aWxlXCI6IHtcbiAgICBcIm91dGJvdW5kIHVubmVnb3RpYXRlZFwiOiA0LFxuICAgIFwib3V0Ym91bmQgL2xpYnAycC9jaXJjdWl0L3JlbGF5LzAuMi4wL2hvcFwiOiAxLFxuICAgIFwib3V0Ym91bmQgL21lc2hzdWIvMS4xLjBcIjogMSxcbiAgICBcImluYm91bmQgL21lc2hzdWIvMS4xLjBcIjogMSxcbiAgICBcImluYm91bmQgL2lwZnMvYml0c3dhcC8xLjIuMFwiOiAxXG4gIH0sXG4gIFwibGlicDJwX2thZF9kaHRfd2FuX3J1bm5pbmdfcXVlcmllc1wiOiAzLFxuICBcImxpYnAycF9rYWRfZGh0X3dhbl9xdWVyeV90aW1lX3NlY29uZHNcIjogMjQ4LFxuICBcImxpYnAycF9rYWRfZGh0X3dhbl9yb3V0aW5nX3RhYmxlX3NpemVcIjogODUsXG4gIFwibGlicDJwX2thZF9kaHRfd2FuX3BpbmdfcXVldWVfc2l6ZVwiOiA5NCxcbiAgXCJsaWJwMnBfa2FkX2RodF93YW5fcGluZ19ydW5uaW5nXCI6IDEwLFxuICBcImxpYnAycF9rYWRfZGh0X2xhbl9ydW5uaW5nX3F1ZXJpZXNcIjogMyxcbiAgXCJsaWJwMnBfa2FkX2RodF9sYW5fcXVlcnlfdGltZV9zZWNvbmRzXCI6IDYxMyxcbiAgXCJsaWJwMnBfa2FkX2RodF9sYW5fcm91dGluZ190YWJsZV9zaXplXCI6IDYzLFxuICBcImxpYnAycF9rYWRfZGh0X2xhbl9waW5nX3F1ZXVlX3NpemVcIjogMCxcbiAgXCJsaWJwMnBfa2FkX2RodF9sYW5fcGluZ19ydW5uaW5nXCI6IDBcbn0pIiwibmFtZSI6ImNsb25lIn1dLCJjb25maWciOnsibmFtZSI6IlNpbXBsZSBleGFtcGxlIHRlc3QiLCJwYXJhbGxlbCI6dHJ1ZSwiZ2xvYmFsVGVzdENvbmZpZyI6eyJkZXBlbmRlbmNpZXMiOltdfSwiZGF0YUNvZGUiOiIifX0=)